client/src/infographic/infographic_link.js

```diff
@@ -15,4 +15,18 @@ const panel_href_template = (
     data_area ? data_area : ""
   }/${panel_key ? SafeJSURL.stringify({ panel_key: panel_key }) : ""}`;
 
-export { infograph_href_template, panel_href_template };
+const infograph_options_href_template = (
```

Comment:
I think you should replace `panel_href_template` with this, they're pretty redundant